Controller/CategoryAdminController.php

```diff
@@ -75,7 +75,7 @@ public function treeAction(Request $request)
             $currentContext = $this->get('sonata.classification.manager.context')->find($context);
         }
 
-        // all root categories
+        // all root categories.
```

Comment:
I suppose this comes from StyleCI?

Reply:
This is needed because otherwise the build will not run, phpunit.xml is not part of the relevant files